ldap/servers/slapd/log.c

```diff
@@ -6861,6 +6861,25 @@ log_flush_buffer(LogBufferInfo *lbi, int log_type, int sync_now, int locked)
                                                         rotationtime_secs);
         }
         log_state = log_refresh_state(log_type);
+
```

Comment:
it looks good but similarly to the log_state you may move that code into a function and call
`fd = log_refresh_filedescriptor(log_type);`

Reply:
> it looks good but similarly to the log_state you may move that code into a function and call `fd = log_refresh_filedescriptor(log_type);`

Implemented